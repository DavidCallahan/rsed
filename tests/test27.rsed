foo = "bar"
print << END
line 1 $foo
line 2
END
print "line 3"
print << "END"r
line 3 $foo
line 4
END
