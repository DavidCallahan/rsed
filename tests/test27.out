line 1 bar
line 2
line 3
line 3 $foo
line 4
